src/Sulu/Bundle/WebsiteBundle/Controller/DefaultController.php

```diff
@@ -45,89 +44,41 @@ public function indexAction(StructureInterface $structure, $preview = false, $pa
 
     /**
      * Creates a redirect for configured webspaces.
+     *
+     * @param Request $request
+     *
+     * @return Response
+     *
+     * @deprecated since 1.2 use SuluWebsiteBundle:Redirect:redirectWebspace instead
      */
     public function redirectWebspaceAction(Request $request)
     {
-        $url = $this->resolveRedirectUrl(
-            $request->get('redirect'),
-            $request->getUri()
-        );
+        @trigger_error('SuluWebsiteBundle:Default:redirectWebspace is deprecated since version 1.2. Use the "SuluWebsiteBundle:Redirect:redirectWebspace" action instead.', E_USER_DEPRECATED);
 
-        return new RedirectResponse($url, 301);
+        return $this->forward(
```

Comment:
like i said i wouldn't add more complexity here and use the functionality of the symfony controller. sometimes this will be deleted and all which uses this should change it to the newer version.